bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/WebexTeamsHandler.java

```diff
@@ -0,0 +1,315 @@
+                }
+            } catch (WebexTeamsApiException e) {
+                logger.warn("Failed to update display name: {}", e.getMessage());
+            }
+
+        }
+    }
+
+    /**
+     * Sends a message to the default room.
+     *
+     * @param msg markdown text string to be sent
+     *
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendMessage(String msg) {
+
+        Message message = new Message();
+        message.setRoomId(config.roomId);
+        message.setMarkdown(msg);
+        logger.debug("Sending message to default room ({})", config.roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Sends a message with file attachment to the default room.
+     *
+     * @param msg markdown text string to be sent
+     * @param attach URL of the attachment
+     *
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendMessage(String msg, String attach) {
+
+        Message message = new Message();
+        message.setRoomId(config.roomId);
+        message.setMarkdown(msg);
+        message.setFile(attach);
+        logger.debug("Sending message with attachment to default room ({})", config.roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Send a message to a specific room
+     * 
+     * @param roomId roomId of the room to send to
+     * @param msg markdown text string to be sent*
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendRoomMessage(String roomId, String msg) {
+        Message message = new Message();
+        message.setRoomId(roomId);
+        message.setMarkdown(msg);
+        logger.debug("Sending message to room {}", roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Send a message to a specific room
+     * 
+     * @param roomId roomId of the room to send to
+     * @param msg markdown text string to be sent
+     * @param attach URL of the attachment
+     * 
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendRoomMessage(String roomId, String msg, String attach) {
+        Message message = new Message();
+        message.setRoomId(roomId);
+        message.setMarkdown(msg);
+        message.setFile(attach);
+        logger.debug("Sending message with attachment to room {}", roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Sends a message to a specific person, identified by email
+     * 
+     * @param personEmail email address of the person to send to
+     * @param msg markdown text string to be sent
+     * @return
```

Comment:
To be completed

Reply:
Not sure how to respond. Maybe it's my incorrect use of english here?

Reply:
Ah never mind.. fixed the return statement.